### DISCUSSION TITLE
Relationships of the class Payment

### DISCUSSION THREAD

**Discussion by eprocurementontology** [eprocurementontology/eprocurementontology#489] [Q&A]
During the third working group meeting, it was explained that a Payment could be made to other parties than Economic Operator. Consequently, which relations should the Class Payment have except the existing relations with Buyer, Economic Operator, Evidence and Monetary Value?

This issue is related to [issue 28](https://github.com/eprocurementontology/eprocurementontology/issues/28) for which the latest version of the conceptual data model included a realtionship between Evidence and Payment.

**Comment by JachymHercher** [comment]
- According to 2014/24/EU Art. 71(3), the buyer can make a payment directly to the seller's subcontractor. 
- Concessions have some specificities as far payments go (see 2014/23/EU Art. 8(3)). For example, a seller can pay the buyer for receiving the concession. 
- When a buyer is buying from a CPB, then a buyer will be paying a buyer.
- In general, all bidders may receive prizes (payments) for participating in the call for competition, not just the winning one. Prizes are more common for design contests and concessions, but in theory can exist also in other procurement regimes.
